horreum-web/src/domain/alerting/Changes.tsx

```diff
@@ -204,16 +205,21 @@ export const flattenNode = (arr : Array<any> | undefined) => {
     return nodeObj;
 }
 
+type ChangesProps = {
+    testID: number
+}
+
 
-export default function Changes() {
+export default function Changes(props: ChangesProps) {
     const { alerting } = useContext(AppContext) as AppContextType;
     const navigate = useNavigate()
     const params = new URLSearchParams(location.search)
-    // eslint-disable-next-line
-    const paramTest = useMemo(() => params.get("test") || undefined, [])
-    const paramFingerprint = params.get("fingerprint")
     const teams = useSelector(teamsSelector)
-    const [selectedTest, setSelectedTest] = useState<SelectedTest>()
+    const newTest = { id: props.testID } as SelectedTest;
+    // Do we need this? the test id is provided by props therefore it should not change
+    const [selectedTest, setSelectedTest] = useState<SelectedTest>(newTest)
```

Comment:
Do we need this? the test id is provided by props therefore it should not change. In fact the function `setSelectedTabs` is not called anywhere

Reply:
I think we can remove this now